CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Project is not versioned, so use date of last upstream commit.
+project(LodePNG LANGUAGES CXX VERSION "2018.10.29")
+
+set(LODEPNG_SRC "lodepng.cpp")
+
+set(LODEPNG_HEADERS "lodepng.h")
+
+add_library(lodepng ${LODEPNG_SRC} ${LODEPNG_HEADERS})
+
+target_include_directories(
+    lodepng
+    PUBLIC
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>"
+)
+
+target_compile_definitions(lodepng PUBLIC "LODEPNG_DEBUG=$<CONFIG:Debug>")
+
+# Introduce variables:
+# * CMAKE_INSTALL_LIBDIR
+# * CMAKE_INSTALL_BINDIR
+include(GNUInstallDirs)
+
+set(GENERATED_DIR "${CMAKE_CURRENT_BINARY_DIR}/generated")
+set(VERSION_CONFIG "${GENERATED_DIR}/${PROJECT_NAME}ConfigVersion.cmake")
+set(PROJECT_CONFIG "${GENERATED_DIR}/${PROJECT_NAME}Config.cmake")
+
+set(CONFIG_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+set(INCLUDE_INSTALL_DIR "include")
```

Comment:
Similar variable defined by GNUInstallDirs:
* https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

You can use `CMAKE_INSTALL_INCLUDEDIR` here, example:
* https://github.com/cgold-examples/fruits/blob/081f76a970d3ebbc8d316ad8ef52994b8b4cad56/CMakeLists.txt#L69

Reply:
Thanks!